webviews/editorWebview/index.css

```diff
@@ -415,6 +415,11 @@ body button .icon {
 	margin-right: 6px;
 }
 
+.review-comment-header .avatar {
+	/* Adds up to 8px together with margin from `.review-comment-header > a` */
+	margin-right: 4px;
+}
+
```

Comment:
This adds up to more than 8px. Note that these are now different spacings. It looks correct in main as is.

From your branch:

<img width="156" alt="CleanShot 2022-10-14 at 16 31 06@2x" src="https://user-images.githubusercontent.com/25163139/195957536-138eade6-709f-428f-9b51-4c6e9e63c9de.png">